pptx_blueprint/__init__.py

```diff
@@ -82,6 +84,20 @@ def _find_shapes_in_slide(slide):
 
         return matched_shapes
 
+    def _get_all_shapes(self) -> Iterable[BaseShape]:
+        # Do we need all the shapes? Perhaps we should filter on tags here.
+        all_shapes = [shape for slide in self._presentation.slides for shape in slide.shapes]
+        return all_shapes
+
+    def _copy_tags_to_name(self) -> None:
+        all_shapes = self._get_all_shapes()
+        # This regex matches on tags
+        regex_tag = re.compile(r'^\s*(\{\w+\})\s*$')
+        for shape in all_shapes:
+            # We only copy contents we recognize as tags
+            if regex_tag.match(shape.text):
```

Comment:
The group is stored in the match object:
~~~
match = regex_tag.match(shape.text)
if match:
    shape.name = match.group(1)
~~~